recursive_retrieval/entity_extraction/entity_extraction.py

```diff
@@ -0,0 +1,40 @@
+from langchain_openai import OpenAI
+from langchain.prompts import PromptTemplate
+from langchain.chains import LLMChain
+from langchain_core.output_parsers import JsonOutputParser
+from typing import List
+import os
```

Comment:
Usually, importing standard library should be top.
You can automatically fix those problems using `yapf` formatter!
```
pip install yapf
yapf -i [files]
```
`-i` means making changes to files in place.

Reply:
Fixed!